demos/calibration/calibration-pubmed-node-classification.ipynb

```diff
@@ -0,0 +1,3065 @@
+{
```

Comment:
<p>May be to create a cell with this command to show it for the users?</p>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/326/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
Before cell 30